aii-pxelinux/src/main/perl/pxelinux.pm

```diff
@@ -494,13 +556,16 @@ sub Unconfigure
     }
 
     my $t = $cfg->getElement (ETH)->getTree;
```

Comment:
maybe not your code, but rename `ETH` to `INTERFACES` and `$t` to `$interfaces` or `$ints`